doc/docs/master/enterprise/auth/index.md

```diff
@@ -0,0 +1,121 @@
+# Configure Access Controls
+
+!!! note
+    Access Controls is an enterprise feature that requires
+    an active enterprise token.
+
+Pachyderm Access Controls enable you to log in to Pachyderm
+with a user configured in a third-party identity management
+platform and operate as that user with the data stored in
+Pachyderm. Pachyderm supports the following identity providers
+with the specified authentication protocols:
+
+- Okta™ with Security Assertion Markup Language(SAML)
+- Keycloak with OpenID Connect (OIDC)
+<!--- Google™ Identity Platform with OIDC -->
```

Comment:
Auth0 and Google have been verified to work

Reply:
added